docs/Staking/Delegates.md

```diff
@@ -1,268 +1,302 @@
-  - Register `stake address` on the blockhain, __with__ key deposit, which is required for participatin in staking and 
-  - Create and submit the __stake delegation certificate__ for the selected pool.
+
+- Create `stake key(s)` for generating the **delegate**'s `reward address(es)` for collecting their `stake rewards`.
+- Create `payment key(s)` for generating the **delegate**'s `payment address(es)` for payments that will participate in stake delegation and also deposititing some funds.
+- Register `stake address` on the blockhain, **with** key deposit, which is required for participation in staking, and
+- Create and submit the **stake delegation certificate** for the selected pool.
 
 #### Create _delegate's_ staking keys and addresses
 
-Stake address, derived from a `stake verification key`, is a simple `reward (account) address` which is not a `payment address` therefore it cannot be used as an output of a transaction.
+A **stake address**, derived from a `stake verification key`, is a simple `reward (account) address` which is not a `payment address` therefore it cannot be used as an output of a transaction.
 In the `cardano-cli` they use a CBOR format for preventing it to be added as the input into a transaction.
 
-> Keep in mind, that only the __reward__ addresses and the __base___, __pointer__ and some __script__ (not available yet) addresses can participate in the staking.
+> Keep in mind, that only the **reward** addresses and the **base\_**, **pointer** and some **script** (not available yet) addresses can participate in the staking.
 
-First, we need to create the relevant `payment` and `stake` keys and the related addresses for the delegate. The payment address for paying the transactions for sending the `registration certificate` (which is just simply the reward account address in some CBOR encoded format) to the chain.
+First, we need to create the relevant `payment` and `stake` keys and the related addresses for the delegate. The payment address will be used for paying the transactions for sending the `registration certificate` (which is just simply the reward account address in some CBOR encoded format) to the chain.
 
-``` bash
+```bash
 # Delegate's staking/acount key, then Staking/Account address
 ############################################################
 mkdir delegate && pushd delegate
-cardano-cli shelley stake-address key-gen --verification-key-file stake.vkey --signing-key-file stake.skey
 
-cardano-cli shelley stake-address build --stake-verification-key-file stake.vkey --testnet-magic 42 > stake.addr
+# Generate the stake address key-pair
+cardano-cli shelley stake-address key-gen \
+--verification-key-file stake.vkey \
+--signing-key-file stake.skey
+
+# Generate the actual stake address with the stake key-pair
+cardano-cli shelley stake-address build \
+--stake-verification-key-file stake.vkey \
+--out-file stake.addr \
+--mainnet
```

Comment:
Perhaps we can add a comment to say use `--mainnet` or `--testnet-magic 1097911063`

Reply:
I'll add a note near the first --mainnet call